scripts/test

```diff
@@ -2,7 +2,10 @@
 set -e
 set -x
 
-unset CC
+if [ "$ARCH" = "mingw32" ] || [ "$ARCH" = "mingw64" ] || [ "$ARCH" = "arm32" ]; then
```

Comment:
Is there a benefit of preferring this over
```suggestion
if [ "$ARCH" = "mingw32" -o "$ARCH" = "mingw64" -o "$ARCH" = "arm32" ]; then
```

Reply:
Thank you! Updated.